File: 125-valid-palindrome/valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string ss = "";
        for(int i=0;i<s.size();i++){
            if(s[i] == ' ' || !isalnum(s[i])){
                continue;
            }
            ss += tolower(s[i]);
        }
            cout << ss << " ";
        int l =0;
        int r = ss.size() - 1;
        while(l<r){
            if(ss[l] != ss[r]){
                return false;
            }

            l++;
            r--;
        }

        return true;
    }
};

File: 125-valid-palindrome/valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:

        def ispali(s):
            l = 0
            r = len(s) - 1

            while(l<=r):
                if s[l] == s[r]:
                    l += 1
                    r -= 1
                else:
                    return False
            return True

        ss = ""

        for ch in s:
            if ch.isalnum():
                ss += ch.lower()
        print(ss)
        return ispali(ss)